src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -556,6 +606,14 @@ def update_container_settings(cmd, resource_group_name, name, docker_registry_se
         update_app_settings(cmd, resource_group_name, name, settings, slot)
     settings = get_app_settings(cmd, resource_group_name, name, slot)
 
+    if bool(multicontainer_config_file) != bool(multicontainer_config_type):
```

Comment:
I thought the `multicontainer_config_file` should be used together with `multicontainer_config_type`, why the opposite?